### DISCUSSION TITLE
Inquiries about how to use SSL in the websocket reverse proxy.

### DISCUSSION THREAD

**Discussion by R3ALW1NNER** [inspircd/inspircd#1951] [Support]
When using websocket, I try to reverse proxy in nginx or haproxy. 

Is there currently no way to use SSL on nginx or haproxy?

I'm trying to SSL on nginx or HAproxy.

**Comment by sadiepowell** [comment]
What error are you actually encountering? 

If you're using a reverse proxy in front of a WebSocket endpoint you should set the proxyranges value in your config so that InspIRCd knows to look up the IP address from the proxy HTTP header.

https://docs.inspircd.org/3/modules/websocket/#websocket

**Comment by R3ALW1NNER** [comment]
Thank you.

I applied the proxy setting and I think it works.

I'll wait and see.
